include/reactor-cpp/assert.hh

```diff
@@ -37,6 +32,33 @@ constexpr bool runtime_assertion = true;
 // NOLINTNEXTLINE(cppcoreguidelines-macro-usage)
 #define reactor_assert(x) assert(x)
 
+#ifdef REACTOR_CPP_USE_BACKTRACE
+
+// NOLINTNEXTLINE
+#include REACTOR_CPP_BACKTRACE_HEADER
+#include <array>
+#include <iostream>
+
+namespace reactor {
+
+constexpr std::size_t MAX_TRACE_SIZE{16};
+
+inline void print_backtrace() {
+  std::array<void*, MAX_TRACE_SIZE> trace{nullptr};
+  int size = backtrace(trace.data(), MAX_TRACE_SIZE);
+  char** messages = backtrace_symbols(trace.data(), size);
+  for (int i{0}; i < size; i++) {
```

Comment:
I know its valid, but this just doesn't look right :D

Reply:
What does not look right?

Reply:
```cpp
int i{0}; 
```